libs/langchain/tests/unit_tests/schema/test_runnable.py

```diff
@@ -409,6 +410,50 @@ async def test_prompt_with_llm(
     )
 
 
+@pytest.mark.asyncio
```

Comment:
Could we add some simple tests with runnable lambdas? 

```
def _simple_recursion(x: int) -> Union[int, Runnable]:
    if x < 10:
        return RunnableLambda(lambda *args: _simple_recursion(x + 1))
    else:
        return x


def test_recursive_lambda() -> None:
    runnable = RunnableLambda(_simple_recursion)
    assert runnable.invoke(5) == 10
```

and something to test recursion limit?

Reply:
Done